pod/perlsub.pod

```diff
@@ -445,8 +445,77 @@ If there are multiple optional positional parameters and not enough
 arguments are supplied to fill them all, they will be filled from left
 to right.
 
-After positional parameters, additional arguments may be captured in a
-slurpy parameter.  The simplest form of this is just an array variable:
+Since Perl 5.43.3, a signature may also declare named parameters.  Like
+positional parameters, these are lexical variables which will contain values
+passed by the caller when the subroutine is invoked.  Whereas positional
+parameters take their values from the corresponding position in the list
+passed by the caller, named parameters are matched by name-value pairs
+from the caller, where the name matches the variable name of the parameter
+but the order of the pairs is not significant.
+
+Named parameters in signatures are currently considered B<experimental>, and
+their behaviour may change in future releases of Perl.  Uses of named
+parameters in a signature will currently emit a warning in the
+C<experimental::signature_named_parameters> category.
+
+Named parameters are declared with a prefixed C<:> symbol in front of the
+variable sigil and name.  This symbol is not part of the variable's name
+within the body of the subroutine.  Named parameters may only be declared
+after any positional parameters; further positional parameters are not
+allowed after any named ones.
+
+    sub foo (:$alpha, :$beta) {
+        print "The value of alpha was $alpha\n";
+    }
+
+To invoke this function, the caller must pass pairs of names and values to
+provide values for the parameters.  Each name must match the name of one of
+the variables, omimtting its leading C<$> sigil.  The order of these values
+is not important; both of the following lines will cause the same behaviour.
```

Comment:
Expected : (colon) rather than . (comma) in the end to introduce the following lines (code samples).